test/functional/test-json.js

```diff
@@ -126,10 +126,10 @@ describe('json', () => {
       const original = {child: {str: 'A', num: 1, bool: true, val: null}};
       const copy = recreateNonProtoObject(original);
       expect(copy).to.deep.equal(original);
-      expect(copy === original).to.be.false;
+      assert(copy !== original);
```

Comment:
Shouldn't this be `expect(copy).to.not.equal(original)`? (Here and everywhere else in the PR)

Reply:
See http://chaijs.com/api/bdd/#method_equal

Reply:
fixed!